main.go

```diff
@@ -33,28 +35,39 @@ func main() {
 
 func run(ctx context.Context) error {
 	signals := make(chan os.Signal, 1)
-	signal.Notify(signals, os.Interrupt, os.Kill)
+	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
 	svcErrors := make(chan error, 1)
 
+	// Read config
+	cfg, err := config.Get()
+	if err != nil {
+		return fmt.Errorf("unable to retrieve configuration, error: %w", err)
+	}
+	log.Info(ctx, "config on startup", log.Data{"config": cfg, "build_time": BuildTime, "git-commit": GitCommit})
```

Comment:
won't this risk emitting private data from the config?

Reply:
The sensitive stuff has the `-` json tag